### PR TITLE
Basic wasm tests - part 1.

### PR DESCRIPTION
**Pull request by poemm**
Everything is ready. I am just not sure how to compute gas balance by hand. Specifically for `wasmDivisionByZero` and `wasmUnreachable`.

### CODE REVIEW

File: src/GeneralStateTestsFiller/stEWASMTests/wasmGrowMemoryFiller.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+           )
+          )
+          ;; store the new memory size
+          (i32.store8
+           (get_local $valAddress)
+           (current_memory)
+          )
+          ;; store memory size at address 0
+          (call $storageStore
+           (get_local $keyAddress)
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmGrowMemoryFiller.yml]: I think you could use `i32.const 0` here or put this comment next to `keyAddress` where it is declared, given it doesn't do an explicit `set_local`.

File: src/GeneralStateTestsFiller/stEWASMTests/wasmUnreachableFiller.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      code: |
+        (module
+         (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+         (memory 1)
+         (data (i32.const 0)  "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; Key
+         (data (i32.const 32) "\cd\ab\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; Value
+         (export "memory" (memory 0))
+         (export "main" (func $main))
+         (func $main
+          ;; this division by zero should trap
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmUnreachableFiller.yml]: I think this comment is wrong.

**Comment by axic** [comment]
Looks good otherwise!

@hugo-dc do you want to check the gas calculations?

File: src/GeneralStateTestsFiller/stEWASMTests/wasmDivisionByZeroFiller.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# Test whether memory grow outputs the size that it grew
```
Comment (hugo-dc) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmDivisionByZeroFiller.yml:1]: did you make any other change after filling the test case, seems this test case needs to be refilled.

File: src/GeneralStateTestsFiller/stEWASMTests/wasmGrowMemoryFiller.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      network:
+        - ALL
+      result:
+        # Balance        SSTORE   TXCOST
+        # 100000000000   - 20000  - 21000   = 99999959000
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          balance: '99999959000'
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage: {
+            0: '0x0000000000000000000000000000000000000003'
```
Comment (hugo-dc) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmGrowMemoryFiller.yml]: For this test case I'm getting different endianess for storage 0: `0x0300000000000000000000000000000000000000000000000000000000000000`

File: src/GeneralStateTestsFiller/stEWASMTests/wasmUnreachableFiller.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
+# Test whether memory grow outputs the size that it grew
```
Comment (hugo-dc) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmUnreachableFiller.yml:1]: This test case needs to be refilled too.

`Test /home/ubuntu/workspace/tests/GeneralStateTests/stEWASMTests/wasmUnreachable.json in wasmUnreachable is outdated. Filler hash is different!`

**Review by hugo-dc — CHANGES_REQUESTED** [review]

**Comment by poemm** [comment]
Changed endian-ness and refilled.

**Review by hugo-dc — APPROVED** [review]

**Review by axic — APPROVED** [review]

**Pull request merged**